### PR TITLE
Fix Typo

### PR DESCRIPTION
**Pull request by rokek**
Correct a misspelling.

### CODE REVIEW

**Comment by nickva** [comment]
Good catch.

Make check fails because a line is longer than 80 characters. Also make sure to rebase on master and make it one commit (I see 4 currently).

File: src/install/troubleshooting.rst [hunk 1]
```diff
@@ -285,7 +285,7 @@ makes use of MD5 hashes for verifying the integrity of data (and not for
 cryptography) and will not run without the ability to use MD5 hashes.
 
 The message below indicates that the operating system is running in "FIPS mode,"
-which among other restrictions does not allow the use of OpenSSL's MD5 funtions:
+which among other restrictions does not allow the use of OpenSSL's MD5 functions:
```
Comment (flimzy) [inline src/install/troubleshooting.rst]: Optional, but while you're correcting the sentence, you could also add commas around the subordinate clause:

> which, among other restrictions, does not...

**Comment by rokek** [comment]
@nickva @flimzy Completed, thank you!

**Comment by nickva** [comment]
Looks great @rokek. Thank you again for your contribution!

**Pull request merged**